ro_help/hub/templates/ngo/detail.html

```diff
@@ -30,6 +31,8 @@ <h2 class="title">
     <p>{{ ngo.description }}</p>
 </div>
 
+{% social_buttons obj=ngo %}
+<br/>
```

Comment:
Same as for using `&nbsp;`.

Reply:
FE not my flavour, but from what I could read around, this is the kind of _acceptable hack_. Happy to change it but can't think of quick decent alternatives from top of my head.